docs/deployments/import-and-export-users.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Import and export users
+description: Learn how to import users to and export users from your Clerk application.
+---
+
+# Import and export users
+
+If you need some guidance or have 200k+ users, please reach out to [support](https://clerk.com/support).
+
+## Import users
+
+You can use the [Backend API](https://clerk.com/docs/reference/backend-api/tag/Users#operation/CreateUser) to create users and import them from your previous provider. 
+
+## Export users
+
+A list of users can be retrieved for use in any exports by calling the backend API found [here](https://clerk.com/docs/reference/backend-api/tag/Users#operation/GetUserList). 
+
+The [`getUserList()`](/docs/references/backend/user/get-user-list) method in the [Clerk Backend SDK](/docs/references/backend/overview) can also be used to retrieve a list of users.
```

Comment:
Do we need to add redirects for this change?

Reply:
https://github.com/clerkinc/clerk-marketing/pull/774